src/test/java/test/assertion/SoftAssertTest.java

```diff
@@ -52,6 +53,16 @@ public void testAssertAllCount() {
     }
   }
 
+  @Test
+  public void testSoftAssertTrace() {
```

Comment:
Can we add a test which does multiple soft assertions? I dont think this test covers the code changes made. We made changes to soft assertion but the test is checking after a hard assert.

Reply:
I thought about that later on in the day, it would be a good idea to add one for multiple soft asserts.

Ohh I see what you mean, a test that checks each soft assert itself.
Let me think on that one, do you have any ideas? I wasn't sure
how to access an error from the m_error list outside of the SoftAssert class
to validate that.

Reply:
You dont need to access the error from outside. You know what all errors were triggered and what messages were associated with them (that would be your expected values), and you just assert them with the message associated with the AssertionError object that gets thrown when you invoke `assertAll()`